spec.md

```diff
@@ -51,9 +55,15 @@ These are the definitions for some of the terms used in this specification:
       Specification, Clause
       4.4](https://spdx.github.io/spdx-spec/v2.3/conformance/#44-standard-data-format-requirements)
       (example: `sbom.spdx.json`).
+    - `COPYING` and `LICENSE`, with or without file extensions. These files
+      typically contain license texts. They are ignored by the REUSE
+      specification for compatibility reasons.
```

Comment:
This is already the first item on the list, albeit more explicit. These items should be merged rather then duplicated.

Reply:
Soft agree, but `COPYING` and `LICENSE` are not Capital L Capital F License Files. License Files here are specifically defined as the files with license texts in the `LICENSES/` directory, which `COPYING` and `LICENSE` are not. In fact, these files needn't even contain license texts—they can be a summary of licensing.

I have moved these two items closer together, and included a more thorough description of `COPYING` and `LICENSE` in the text body.